src/app/admin/components/equipment/equipment.component.ts

```diff
@@ -0,0 +1,164 @@
+import { ChangeDetectionStrategy, Component, ElementRef, Inject, OnInit, ViewChild } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { Equipment } from '@app/catalog/models/equipment';
+import { BaseKind, EquipmentKind, EquipmentOptions, PetSize } from '@app/shared/models/management';
+import { AbstractControl, FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { TechnicalIssues } from '@app/shared/types/technical-issues';
+import { ValidationService } from '@app/shared/services/validation/validation.service';
+import { NewEquipment } from '@app/shared/models/equipment';
+import { EquipmentModal } from '@app/admin/constants/equipment-modal.enum';
+import { ErrorOptions } from '@app/shared/types';
+import { EquipmentMOdalData } from '@app/admin/types/equipoment-modal-data';
+import { maxInventoryNumber } from '@app/admin/constants/max-inventory-number';
+import { maxCompensationCost } from '@app/admin/constants/max-compensation-cost';
+
+@Component({
+  selector: 'lc-equipment',
+  templateUrl: './equipment.component.html',
+  styleUrls: ['./equipment.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class EquipmentComponent implements OnInit {
```

Comment:
maybe better to call it EquipmentModalComponent or EquipmentFormComponent? EquipmentComponent sounds like a view page component rather than creating/editing component

Reply:
fixed